packages/dashmate/README.md

```diff
@@ -238,9 +238,27 @@ DESCRIPTION
   Reset node data
 ```
 
+With the hard reset mode enabled, the corresponding config will be reset as well. To proceed, running the node [setup](#setup-node) is required.
+
 To reset a node:
 ```bash
 $ dashmate reset
+=======
+#### Hard reset
+
+``dashmate reset --hard``
+
+With the hard reset mode enabled, the corresponding config will be reset as well. This command cleans up all related containers and volumes. To proceed, running the node [setup](#setup-node) is required.
+
+#### Manual reset
+
+Manual reset is used when local setup corrupts and hard reset does not fix it. This could happen, when dashmate configuration becomes incompatible after a major upgrade, making you unable to execute any commands.
+
+```bash
+docker stop $(docker ps -q)
+docker system prune
+docker volume prune
+rm -rf ~/.dashmate/
 ```
```

Comment:
There appear to be some formatting issues here (possibly from a merge conflict at some point). I _think_ it's supposed to look something like the following:

Note: the preview looks a weird to me too but I think it could be due to the use of "\`\`\`" characters. Maybe merge the comment and we can re-review and see if it looks right.

```suggestion
```bash
$ dashmate reset
```

#### Hard reset

`dashmate reset --hard`

With the hard reset mode enabled, the corresponding config will be reset as well. This command cleans up all related containers and volumes. To proceed, running the node [setup](#setup-node) is required.

#### Manual reset

Manual reset is used when local setup corrupts and hard reset does not fix it. This could happen, when dashmate configuration becomes incompatible after a major upgrade, making you unable to execute any commands.

```bash
docker stop $(docker ps -q)
docker system prune
docker volume prune
rm -rf ~/.dashmate/
```

Reply:
Sorry @thephez but after merging your suggestions that part of documentation was totally messed up. I brought back previous version with some improvements.